packages/contracts-bedrock/src/L1/OptimismPortal.sol

```diff
@@ -21,6 +21,23 @@ import { Constants } from "src/libraries/Constants.sol";
 ///         and L2. Messages sent directly to the OptimismPortal have no form of replayability.
 ///         Users are encouraged to use the L1CrossDomainMessenger for a higher-level interface.
 contract OptimismPortal is Initializable, ResourceMetering, ISemver {
+    /// @notice Error for when a deposit or withdrawal is to a bad target.
+    error BadTarget();
```

Comment:
definitely would rather these be in a shared lib than copied into both

Reply:
Ok sounds good to me, i agree. Any preference on the library name?